src/containers/Journal/index.js

```diff
@@ -1,8 +1,114 @@
-import React from 'react';
-import UnderConstruction from '../UnderConstruction';
+import React, { Component } from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import { Icon, Popconfirm } from 'antd';
+import BackButton from '../../components/BackButton';
+import JournalComponent from '../../components/Journal';
 
-const Journal = props => {
-  return <UnderConstruction {...props} />;
+import './journal.css';
+
+class Journal extends Component {
+  state = {
+    journal: {
```

Comment:
Put the fake data in another file and import it then spread it here 
```suggestion
    journal: [...data]
```
It will be more readable I guess :nerd_face: